pegasus/sites.v3/code.org/public/js/mobile-footer.js

```diff
@@ -6,10 +6,13 @@ $(function () {
     $('#down').toggle();
   });
 
-  document.querySelector( '#footer-toggle' )
-    .addEventListener( "click", function () {
-    this.classList.toggle( 'active' );
-  });
+  var footerToggle = document.querySelector( '#footer-toggle' );
```

Comment:
Yes, when there are all of the links, they are in a sort of drawer that opens and closes.